spi-fly/spi-fly-dynamic-bundle/pom.xml

```diff
@@ -51,8 +51,8 @@
     <dependencies>
         <dependency>
             <groupId>org.ow2.asm</groupId>
-            <artifactId>asm-debug-all</artifactId>
-            <version>5.0.3</version>
+            <artifactId>asm-all</artifactId>
+            <version>6.0_ALPHA</version>
```

Comment:
I would prefer with waiting to move up to 6.0 until ASM 6.0 is not in alpha any more.